cpu/samd5x/periph/can.c

```diff
@@ -565,11 +581,12 @@ static int _set_filter(candev_t *candev, const struct can_filter *filter)
         }
 
         DEBUG("Standard Filter to add at idx = %d\n", idx);
-        dev->msg_ram_conf.std_filter[idx].SIDFE_0.bit.SFEC = filter_conf;
         /* For now, only CLASSIC filters are supported */
-        dev->msg_ram_conf.std_filter[idx].SIDFE_0.bit.SFT = CANDEV_SAMD5X_CLASSIC_FILTER;
-        dev->msg_ram_conf.std_filter[idx].SIDFE_0.bit.SFID1 = filter->can_id & CAN_SFF_MASK;
-        dev->msg_ram_conf.std_filter[idx].SIDFE_0.bit.SFID2 = filter->can_mask & CAN_SFF_MASK;
+        dev->msg_ram_conf.std_filter[idx].SIDFE_0.reg = CAN_SIDFE_0_SFEC(filter_conf)
+                                                        | CAN_SIDFE_0_SFT(CANDEV_SAMD5X_CLASSIC_FILTER)
+                                                        | CAN_SIDFE_0_SFID1(filter->can_id & CAN_SFF_MASK)
+                                                        | CAN_SIDFE_0_SFID2(filter->can_mask & CAN_SFF_MASK);
+
```

Comment:
```suggestion
        dev->msg_ram_conf.std_filter[idx].SIDFE_0.reg = CAN_SIDFE_0_SFEC(filter_conf)
                                                      | CAN_SIDFE_0_SFT(CANDEV_SAMD5X_CLASSIC_FILTER)
                                                      | CAN_SIDFE_0_SFID1(filter->can_id & CAN_SFF_MASK)
                                                      | CAN_SIDFE_0_SFID2(filter->can_mask & CAN_SFF_MASK);

```

IMO this looks better. But let's not argue about style. If you agree, please squash directly after applying. If you don't agree, just keep it as it is.

Reply:
Fixed.